src/Hubbup.Web/Diagnostics/Metrics/IMetricsBuilder.cs

```diff
@@ -0,0 +1,10 @@
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Hubbup.Web.Diagnostics.Metrics
+{
+    // Marker interface to hang .Add[MetricSinkName] extensions off of.
```

Comment:
Not *really* a marker interface if it has an API on it, even if it's trivial. Whatev's.

Reply:
Fair point. I mean it's a marker in that it's just an `IServiceProvider` with a different type name, like `ILoggerBuilder`.